assets/svgbob_worker.js

```diff
@@ -0,0 +1,150 @@
+let importObject = {
+  env: {
+    abort: () => console.log("abort")
+  }
+};
+
+
+let wasm;
+let wasmModuleUrl = "https://unpkg.com/svgbob-wasm@0.4.1/svgbob_wasm_bg.wasm";
+
+if (WebAssembly.instantiateStreaming) {
+  WebAssembly.instantiateStreaming(
+    fetch(wasmModuleUrl),
+    importObject
+  ).then((response) => {
+    wasm = response.instance.exports;
+  })
```

Comment:
I'm not sure if this case will ever happen. I suppose this is intended to be a polyfill for `instantiateStreaming`?

The else case is kind of funny. I suppose, it's doing the same thing in a different way.

Nothing that needs to be changed, though.

Reply:
Yeah I was worried this was blowing stuff up turns out it probably was not: 

https://caniuse.com/?search=instantiatestreaming

Looks green enough to me, I may remove ... not sure.